pytest_mh/utils/fs.py

```diff
@@ -230,6 +231,23 @@ def exists(self, path: str) -> bool:
 
         return False
 
+    def selinux_context(self, path: str) -> SELinuxContext | None:
+        """
+        Checks SELinux file context.
```

Comment:
Maybe 'Reads' or 'Gets'? 'Checks' is misleading here, imo.

Reply:
Changed to 'Gets'